### DISCUSSION TITLE
Provide separate bundled CSS files (especially for different themes)

### DISCUSSION THREAD

**Discussion by frnora** [clientIO/joint#2281] [Ideas]
Hi,

Would it be possible to a have separate/split `rappid.css` in the jointjs+ package? Right now it contains all the themes which I believe is an overkill for most of the users (how many apps actually need them?).

It would be nice to have something like:

- rappid-all.css (if somebody really needs all the functionality/themes)
- rapid-base.css (layouting stuff)
- rappid-theme-default.css (themes)
- ...etc.

I know this can be achieved just bundling the provided css files but it would be really nice to have it straight of the the zip file or have some kind of "builder" for it, possibly even excluding classes which are not used. Thank you.

**Comment by zbynekstara** [comment]
Hi @frnora, thank you for the suggestion. This feature will be part of the JointJS+ patch 3.7.2 coming very soon.

Just a small modification - for backwards compatibility, we adjusted the naming slightly from what you suggest:
- the current .css file with everything inside will keep its old name `rappid.css`
- the new base .css file will have the name `rappid.base.css`
- the new themes .css files will have the naming scheme `rappid.theme._.css`
